### PR TITLE
chore(concept): rename propery from alternative to hasAlternatives

### PR DESCRIPTION
**Pull request by mkanal**
chore(concept): rename propery from alternative to hasAlternatives

### CODE REVIEW

File: docs/concept/#212-Update-of-aspect-model-SingleLevelBomAsBuilt-2_0_0/#212-Update-SingleLevelBomAsBuilt-2-0-0.md [hunk 1]
```diff
@@ -97,8 +97,10 @@ flowchart LR
     s1 --> SingleLevelBomAsBuilt2
     SingleLevelBomAsBuilt2(SingleLevelBomAsBuilt) --> s4((AAS C-X 2 \n hasAlternatives=false))
     s4 --> SerialPart3(SerialPart)
+   s4 --> SerialPart5(SingleLevelBomAsBuilt2.0.0)
     SingleLevelBomAsBuilt2 --> s5((AAS C-X 3 \n hasAlternatives=false))
     s5 --> SerialPart2(SerialPart)
+    s5 --> SerialPart4(SingleLevelBomAsBuilt2.0.0)
```
Comment (ds-jhartmann) [inline docs/concept/#212-Update-of-aspect-model-SingleLevelBomAsBuilt-2_0_0/#212-Update-SingleLevelBomAsBuilt-2-0-0.md:103]: name mismatch: SerialPart or SingleLevelBomAsBuilt2.0.0 ?

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: docs/concept/#212-Update-of-aspect-model-SingleLevelBomAsBuilt-2_0_0/#212-Update-SingleLevelBomAsBuilt-2-0-0.md (hunk 1 above)
Comment (mkanal) [inline docs/concept/#212-Update-of-aspect-model-SingleLevelBomAsBuilt-2_0_0/#212-Update-SingleLevelBomAsBuilt-2-0-0.md:103]:
```suggestion
    s5 --> SingleLevelBomAsBuilt4(SingleLevelBomAsBuilt2.0.0)
```

File: docs/concept/#212-Update-of-aspect-model-SingleLevelBomAsBuilt-2_0_0/#212-Update-SingleLevelBomAsBuilt-2-0-0.md [hunk 2]
```diff
@@ -97,8 +97,10 @@ flowchart LR
     s1 --> SingleLevelBomAsBuilt2
     SingleLevelBomAsBuilt2(SingleLevelBomAsBuilt) --> s4((AAS C-X 2 \n hasAlternatives=false))
     s4 --> SerialPart3(SerialPart)
+   s4 --> SerialPart5(SingleLevelBomAsBuilt2.0.0)
```
Comment (mkanal) [inline docs/concept/#212-Update-of-aspect-model-SingleLevelBomAsBuilt-2_0_0/#212-Update-SingleLevelBomAsBuilt-2-0-0.md:100]:
```suggestion
   s4 --> SingleLevelBomAsBuilt5(SingleLevelBomAsBuilt2.0.0)
```

**Review by ds-jhartmann — APPROVED** [review]
LGTM

**Pull request merged**